target/linux/ramips/mt7620/base-files/etc/board.d/02_network

```diff
@@ -130,6 +130,13 @@ ramips_setup_interfaces()
 		ucidef_add_switch "switch0" \
 			"1:lan" "2:lan" "3:lan" "5:lan" "0:wan" "6@eth0"
 		;;
+	dlink,dwr-961-a1)
+		ucidef_add_switch "switch0" \
+			"2:lan:1" "3:lan:2" "4:lan:3" "5:lan:4" "1:wan" "0@eth0"
+		ucidef_add_switch "switch1" \
+			"5:lan" "6@eth0"
```

Comment:
I've tested this setup on a mt7620+mt7530 device and realized that an internal switch setup is helpful for luci.
But port 5 is connected to the external switch so it is used for both LAN and WAN port, right? Then I feel `"5:lan"` is not a good port name...

Reply:
Honestly, I have no idea. If we introduce new role, it will add a new vlan to eth0. So I think, using 'lan' role is the best way.

Someday the problem will gone, if mt7620 receive DSA support.

Reply:
Okay, fair enough. I don't have any good alternative either.